app/controllers/concerns/conditional_user_mailer.rb

```diff
@@ -1,35 +1,27 @@
+# frozen_string_literal: true
+
 module ConditionalUserMailer
-  extend ActiveSupport::Concern
 
-  # Adds following methods as class methods for the class that include this module
-  included do
```

Comment:
`ActiveSupport::Concern` was not being used correctly here. The included block is a callback, and is not necessary to include methods within another class (that's the default behaviour of Modules in Ruby).

Please read the documentation for [ActiveSupport::Concern](https://api.rubyonrails.org/v5.2/classes/ActiveSupport/Concern.html) for tips on correct usage.